### ISSUE TITLE
Blender and Eclipse PyDev connecting issue: Blender paused itself but nothing happened in Eclipse PyDev server

### ISSUE THREAD

**Issue by YuzhouGuo** [TylerGubala/blenderpy#87]
### The Bug
I followed the https://wiki.blender.org/wiki/Tools/Debugging/Python_Eclipse post and set up the PyDev package in Eclipse successfully. I pasted the exact same code from the post before my own code and started the PyDev debug server before running the Blender script as well.

**The Blender did pause and disabled input (the cursor became a loading icon in this case). However, when I switch to Eclipse manually, nothing really happened. I didn't see my python file opened anywhere, nothing highlighted or anything else.**

Thank you, any help will be much appreciated!

### Expected behavior
According to the Blender Debugging post, we should expect something like this:

_Run the script in Blender - at this point Blender will hang (stop responding to input).
Switch to Eclipse.
The file you are debugging should be open, with the line it has stopped at highlighted_

### What I installed and used
 - Python 3.7
 - future-fstrings 1.2.0
 - pip 21.0.1
 - setuptools 40.8.0
 - six 1.6.0
 - PyDev package
 - Launched Blender from Eclipse (installed Blender as external)

**Comment by TylerGubala** [comment]
How did you set up your Eclipse environment? 

I downloaded the Eclipse installer. Then I go to https://www.pydev.org/download.html

But the download links at the bottom are broken.

**Comment by YuzhouGuo** [comment]
Yeah that's how I installed my Eclipse as well. I downloaded the 2021/03 version of Eclipse and went to "Install new software" and installed PyDev using the link (https://www.pydev.org/download.html). Didn't try it out with other exe. though, the only thing I have connected with PyDev is Blender so not sure if the issue comes from the Pydev or Blender side.
